google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploymentConfigurationPanel.java

```diff
@@ -123,6 +128,12 @@ public void applyEditorTo(@NotNull AppEngineDeploymentConfiguration configuratio
     configuration.setPromote(promoteCheckbox.isSelected());
     configuration.setCloudProjectName(projectSelector.getText());
     configuration.setStopPreviousVersion(stopPreviousVersionCheckbox.isSelected());
+    configuration.setDeployAllConfigs(deployAllConfigsCheckbox.isSelected());
```

Comment:
I wanted to separate the responsibility of setting config options to the panel that owns the UI elements; in this case, the field is in the common panel. If we want to restrict it to standard editors only, IMO the UI should also move out of the common panel and into the standard editor. WDYT?